apps/tlon-mobile/src/db/queries.ts

```diff
@@ -0,0 +1,140 @@
+import { UpdateMode, realm } from './realm';
+import type {
+  Channel,
+  Group,
+  SchemaMap,
+  SchemaModel,
+  SchemaName,
+  SchemaValue,
+} from './schemas';
+
+// Model queries
+
+export interface GroupQuerySettings {
```

Comment:
Just a note: I ultimately want these to query functions to follow a more consistent pattern, but haven't found the right answer yet.